ui/packages/catalog-ui-search/src/main/webapp/react-component/layer-item/presentation/interactions.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import { hot } from 'react-hot-loader'
+import { PresentationProps, IsButton, HighlightBehavior } from '.'
+
+const Interactions = styled.div`
+  text-align: right;
+`
+
+const InteractionsButton = styled.button`
+  ${props => IsButton(props.theme)};
+  ${props => HighlightBehavior({ initialOpacity: props.theme.minimumOpacity })};
+  width: ${props => props.theme.minimumButtonSize};
+  height: ${props => props.theme.minimumButtonSize};
+  vertical-align: top;
+`
+
+const Warning = styled(InteractionsButton)`
+  display: inline-block;
+  color: ${props => props.theme.warningColor};
+  cursor: default;
+`
+/* stylelint-disable block-no-empty */
```

Comment:
can we remove this?

Reply:
It won't let empty styled components compile without this :/

Reply:
Why does it need to be styled?

Reply:
Just a preference to name the button instead of using the base InteractionsButton